### DISCUSSION TITLE
Power Pages, Using CLI tool to exclude web pages deployment from Dev to QA

### DISCUSSION THREAD

**Discussion by Talonx32** [microsoft/powerplatform-build-tools#998] [Q&A]
Hello, I'm hoping someone can answer this question. In my dev environment my developers are creating web pages including pages that are for testing purposes. I want to exclude some of those pages when moving from dev to QA. Is there a way using the CLI pac command to exclude pages that I don't want to deploy to our QA environment?

**Comment by jbuenoliveira** [comment]
No, that option is not available. I suggest maintaining a solution with the expected components for moving across environments and a workbench solution with the components that are not yet ready to move.
An alternative approach would be to create a custom pipeline (w/ parameters to control the behavior) that exports the solution and then uses PowerShell to remove specific components either from the artifact or directly from the Git commit to the repository.
